public/video-ui/src/components/VideoEdit/VideoEdit.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import VideoTitleEdit from './formComponents/VideoTitle';
+
+
+export default class VideoEdit extends React.Component {
+
+  constructor(props) {
+    super(props);
+  }
+
+  render () {
+    if (!this.props.video) {
+      console.log('VideoEdit loaded without video provided');
+      return false;
+    }
+
+    if (this.props.video.data) {
```

Comment:
Could this be combined into the above if statement so no need to wrap main the actual return in an if statement